src/footer/footer.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './footer.css'
+import { Link } from 'react-router-dom';
+
+const footerLinks = [
+    ['Home', 'Shop', 'About', 'Contact'],
+    ['Payment Options', 'Returns', 'Privacy Policies']
+]
+
+const FooterLinksList = ({ elem }) => {
+    return elem.map((link) => {
+        return (
+            <Link className='footer_link' to=''>
+                {link}
+            </Link>
+        )
+    })
+}
+
+
+
+const Footer = () => {
+    return (
+        <footer className='footer'>
+            <div className='links'>
+                <div className='funiro_col furniro_col_first'>
+                    <h3 className='funiro'><img src="/photos/funiro..svg" alt="funiro." /></h3>
+                    <p className='funiro_lorem'>400 University Drive Suite 200 Coral Gables,
+                        <br /> FL 33134 USA</p>
+                </div>
+                <div className='funiro_col furniro_col_second'>
+                    <h3 className='links_title'>Links</h3>
+                    {<FooterLinksList elem={footerLinks[0]} />}
```

Comment:
this is not implemented how I showed you. additionally `elem` prop name is not informative, use something meaningful for your props, e.g. `link`